docs/api/navigators/DrawerNavigator.md

```diff
@@ -128,6 +128,7 @@ const styles = StyleSheet.create({
 
 ### `contentOptions` for `DrawerView.Items`
 
+- `routes` - an array of routes to use in place of `navigation.state.routes`
```

Comment:
I would call them 'items'. Of course the routes are currently used as the source of item data, but I see you want to use an alternative list of items other than the routes.